rfcs/text/0010-email.md

```diff
@@ -77,6 +77,43 @@ Stage 1: Provide a high-level description of example sources of data. This does
 <!--
 Stage 2: Included a real world example source document. Ideally this example comes from the source(s) identified in stage 1. If not, it should replace them. The goal here is to validate the utility of these field changes in the context of a real world example. Format with the source name as a ### header and the example document in a GitHub code block with json formatting.
 -->
+```json
```

Comment:
It looks like these are Microsoft 365 example source events for one successful and one undeliverable message, but can we add some labels making that clearer here?

Reply:
++